types/react-overlays/lib/ModalManager.d.ts

```diff
@@ -1,10 +1,17 @@
-import * as Modal from './Modal';
+import * as Modal from "./Modal";
 
 declare class ModalManager {
-  constructor(opts?: { hideSiblingNodes?: boolean; handleContainerOverflow?: boolean });
-  add(modal: Modal, container: any, className?: string): number;
-  remove(modal: Modal): void;
-  isTopModal(modal: Modal): boolean;
+	constructor(opts?: {
+		hideSiblingNodes?: boolean;
+		handleContainerOverflow?: boolean;
+	});
+	isContainerOverflowing(modal: Modal): boolean;
+	containerIndexFromModal(modal: Modal): number;
+	setContainerStyle(containerState: any, container: any): void;
```

Comment:
`containerState` seems to be a type `{ modals: Modal[]; style: CSSStyleDeclaration; classes: string[]; overflowing: boolean;  }`... not sure if it's worth defining here since this seems to be an internal piece of the library, but if we're going to put a definition might as well make it complete.